src/main/java/com/google/cloud/tools/ide/login/GoogleLoginState.java

```diff
@@ -390,8 +354,9 @@ private void persistCredentials() {
 
     for (Account account : accountRoster.getAccounts()) {
       OAuthData oAuthData = new OAuthData(account.getAccessToken(), account.getRefreshToken(),
-          account.getEmail(), oAuthScopes, account.getAccessTokenExpiryTime());
+          account.getEmail(), account.getName(), account.getAvatarUrl(),
+          oAuthScopes, account.getAccessTokenExpiryTime());
       authDataStore.saveOAuthData(oAuthData);
```

Comment:
Hmm... you're right. I see two options for now:

- Rename `OAuthData` to `OAuthDataStore`. Maybe it makes sense to remove `OAuthData` and use `Account` instead. They are very similar. `OAuthDataStore` could be renamed to `AccountStore`. Or `AccountRosterStore`.
-  Retain current `OAuthData` and fetch avatar URLs and names whenever `GoogleLoginState` loads stored credentials.